### DISCUSSION TITLE
Are there ways to tweak the Review Duplicaties?

### DISCUSSION THREAD

**Discussion by elsingaa** [immich-app/immich#12852] [Q&A]
I have a _lot_ of duplicate files. Years of taking photos and backing them up like crazy does that. I have folders with names like 'uploaded', 'to process', 'unknown', etc. 

The Review Duplicates utility is one of my newest friends. However, it often picks the 'wrong' duplicate to keep, I would select other one. I now go into the view of the file and look at folder location of the file and select accordingly.

Is there a way to tweak the duplicates process so that I can give it (negative) words to base its Keep or Trash selection on?

**Comment by mertalev** [comment]
We intend to have different options to customize how the tool picks assets, including quality (based on a machine learning model), oldest/newest and possibly certain other metadata.

**Reply by Voxel07** [reply]
It would be awesome if it would be possible to have the possibility to transfer the album from the "trash" photo to the "keep" with a single buttonlike

Here I would like to keep the "keep" image but want it in the same album as the "trash" image.
![image](https://github.com/user-attachments/assets/7d8394c0-afc3-4379-95f9-5ef37b45de9f)
